content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Veneers"
+linkTitle: "Veneers"
+weight: 5
+date: 2022-06-30
+---
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions.  FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/).  Unfortunately, FBCs can get complex, especially as the number of releases and dependencies scale.
```

Comment:
There are a lot of double-white-spaces at the beginning of every sentence in this doc. Not sure why. Is this for visual enhancement? Otherwise, please clear them up as it creates strange spacing while reading the doc.

Reply:
You will notice the larger space at the beginning of every sentence. It messes up the even spacing between every word. Unless there is a reason behind that extra space, I would recommend removing it.